src/modules/User/resolvers.ts

```diff
@@ -0,0 +1,38 @@
+import { queryField, mutationField, nonNull } from 'nexus/dist';
+import { UserInterface } from './model';
+
+export const me = queryField('me', {
+  type: 'User',
+  resolve: (_, __, ctx) => {
+    if (!ctx.authVerificationResult.userId) {
+      throw new Error('Unexpected middleware behaviour.');
+    }
+    return ctx.prisma.user.findUnique({
+      where: {
+        id: ctx.authVerificationResult.userId,
+      },
+    });
+  },
+});
+
+export const signIn = mutationField('signIn', {
+  type: 'AuthPayload',
+  args: {
+    user: nonNull('UserNamePass'),
+  },
+  resolve: async (_parent, { user: userNamePass }, ctx) => {
+    const userInterface = new UserInterface(ctx.prisma.user);
```

Comment:
Please provide user interface inside context instead of recreating it every time. Probably a cleaner way would be to use `$extend` functionality [described here](https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions/model)